boards/arm/stm32l562e_dk/stm32l562e_dk_defconfig

```diff
@@ -21,6 +21,7 @@ CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
 # PLL configuration
 CONFIG_CLOCK_STM32_PLL_SRC_MSI=y
 CONFIG_CLOCK_STM32_MSI_RANGE=6
+CONFIG_CLOCK_STM32_MSI_PLL_MODE=y
```

Comment:
This symbol has a dependency on CONFIG_CLOCK_STM32_LSE=y. It should be added

Reply:
could be 
```
config CLOCK_STM32_MSI_PLL_MODE
	default y
	depends on STM32_LPTIM_CLOCK_LSE
```
in the soc/arm/st_stm32/stm32l5/Kconfig.defconfig.series